pages/tidal tools/analyze_database.md

```diff
@@ -95,6 +99,10 @@ databases:
 
 {% include note.html content="It is best to use quotations, either double or single, around the values in the configuration file. To avoid special characters, : { } [ ] , & * # ? | - < > = ! % @ \ \n from being interpreted. Single quotes are safest, if the value has a single quote within it, you can include it by using a two single quotations, ie. `'my''string'` - will become `my'string`." %}
 
+{% include tip.html content="Are you analyzing an Oracle database and need to
+only use features that are including the Oracle Standard Edition (SE) license?
```

Comment:
“to only use features” or “to use only features”?

> features that are including the Oracle Standard Edition (SE) license

I think it should be “features … included in the … license”. WDYT?

Maybe let's rewrite this sentence into something like the following:

**Oracle Standard Edition (SE) user? Check out the advanced configuration section below to know how to perform your database analysis using only those features which are permitted by the terms of your Oracle license.**

WDYT?